python-sdk/nuscenes/can_bus/README.md

```diff
@@ -0,0 +1,132 @@
+
+## CAN bus messages
+Here we list the raw CAN bus messages.
+We store each type of message in a separate file for each scene (e.g. `scene-0001_ms_imu.json`).
+Messages are stored in chronological order in the above file. 
+Each message has the following field:
+- utime: \[1\] The integer timestamp in microseconds that the actual measurement took place (e.g. 1531883549954657).
+For the *Zoe Sensors* and *Zoe Vehicle Info* messages this info is not directly available and is therefore replaced by the timestamp when the CAN bus message was received.
+
+### IMU 
+(100Hz, *scene_%04d_ms_imu.json*)
+
+- linear_accel: \[3\] Acceleration vector (x, y, z) in the IMU frame in m/s/s.
+- q: \[4\] Quaternion that transforms from IMU coordinates to a fixed reference frame. The yaw of this reference frame is arbitrary, determined by the IMU. However, the x-y plane of the reference frame is perpendicular to gravity, and z points up. 
+- rotation_rate: \[3\] Angular velocity in rad/s around the x, y, and z axes, respectively, in the IMU coordinate frame.
+
+### Pose
+(50Hz, *scene_%04d_pose.json*)
+
+The current pose of the ego vehicle, sampled at 50Hz.
+- accel: \[3\] Acceleration vector in the ego vehicle frame in m/s/s.
+- orientation: \[4\]  The rotation vector in the ego vehicle frame.
+- pos: \[3\] The position (x, y, z) in meters in the ego vehicle frame. This should be identical to the [nuScenes ego pose](https://github.com/nutonomy/nuscenes-devkit/blob/master/schema.md#ego_pose), but sampled at a higher frequency.
+- rotation_rate: \[3\] The angular velocity vector of the vehicle in rad/s.  This is expressed in the ego vehicle frame.
+- vel: \[3\] The velocity in m/s, expressed in the ego vehicle frame.
+ 
+### Steer Angle Feedback
+(100Hz, *scene_%04d_steeranglefeedback.json*)
+
+- value: \[1\] Steering angle feedback in radians in range \[-7.7, 6.3\]. 0 indicates no steering, positive values indicate right turns, negative values left turns.
+
+### Vehicle Monitor
+(2Hz, *scene_%04d_vehicle_monitor.json*)
+
+- available_distance: \[1\] Available vehicle range given the current battery level in kilometers.
+- battery_level: \[1\] Current battery level in range \[0, 100\].
+- brake: \[1\] Braking pressure in bar. An integer in range \[0, 126\]. 
+- brake_switch: \[1\] Brake switch as an integer, 1 (pedal not pressed), 2 (pedal pressed) or 3 (pedal confirmed pressed).
+- gear_position: \[1\] The gear position as an integer, typically 0 (parked) or 7 (driving).
+- left_signal: \[1\] Left turning signal as an integer, 0 (inactive) or 1 (active).
+- rear_left_rpm: \[1\] Rear left brake speed in revolutions per minute.
+- rear_right_rpm: \[1\] Rear right brake speed in revolutions per minute.
+- right_signal: \[1\] Right turning signal as an integer, 0 (inactive) or 1 (active).
+- steering: \[1\] Steering angle in degrees at a resolution of 0.1 in range \[-780, 779.9\].
+- steering_speed: \[1\] Steering speed in degrees per second.
```

Comment:
Is this radians / second ?

Reply:
Nope, degrees / second. I've added the min and max to show that it is signed.